configure.ac

```diff
@@ -74,7 +74,7 @@ AM_CONDITIONAL([OS400],    [AS_CASE([$host_os],[os400],         [true], [false])
 AM_CONDITIONAL([SUNOS],    [AS_CASE([$host_os],[solaris*],      [true], [false])])
 AM_CONDITIONAL([WINNT],    [AS_CASE([$host_os],[mingw*],        [true], [false])])
 AS_CASE([$host_os],[mingw*], [
-    LIBS="$LIBS -lws2_32 -lpsapi -liphlpapi -luserenv -luser32 -ldbghelp -lole32 -luuid -lshell32"
```

Comment:
Can you also drop the flag from CMakeLists.txt? Other than that LGTM.

Reply:
[Done](https://github.com/libuv/libuv/pull/4261/commits/02a282762c01638d571f22f04a2af0993138898a), thank you!